engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java

```diff
@@ -150,4 +171,34 @@ public boolean reallyCallCamel(JsonObject body, UUID historyId, String accountId
 
     }
 
+    private boolean callOpenBridge(JsonObject body, UUID id, String accountId, CamelProperties camelProperties) {
+
+        Map<String, String> extras = camelProperties.getExtras();
+
+        Map<String, Object> ce = new HashMap<>();
+
+        ce.put("id", id);
+        ce.put("source", "notifications"); // Source of original event?
+        ce.put("specversion", "1.0");
+        ce.put("type", "myType"); // Type of original event?
+        ce.put("rhaccount", accountId);
+        ce.put(PROCESSORNAME, extras.get(PROCESSORNAME));
+        // TODO add dataschema
+
+        LOGGER.infof("Sending Event with id(%s) for processor with name %s and id=%s\n",
```

Comment:
Can we get rid of the new line char?
```suggestion
        LOGGER.infof("Sending Event with id(%s) for processor with name %s and id=%s",
```